packages/cinterop/build.gradle.kts

```diff
@@ -85,23 +87,23 @@ android {
 }
 
 val nativeLibraryIncludesMacosUniversal = listOf(
-    "-include-binary", "$rootDir/../external/core/capi_macos_universal/src/realm/object-store/c_api/librealm-ffi-static-dbg.a",
-    "-include-binary", "$rootDir/../external/core/capi_macos_universal/src/realm/librealm-dbg.a",
-    "-include-binary", "$rootDir/../external/core/capi_macos_universal/src/realm/parser/librealm-parser-dbg.a",
-    "-include-binary", "$rootDir/../external/core/capi_macos_universal/src/realm/object-store/librealm-object-store-dbg.a"
+    "-include-binary", "$rootDir/external/core/capi_macos_universal/src/realm/object-store/c_api/librealm-ffi-static-dbg.a",
+    "-include-binary", "$rootDir/external/core/capi_macos_universal/src/realm/librealm-dbg.a",
+    "-include-binary", "$rootDir/external/core/capi_macos_universal/src/realm/parser/librealm-parser-dbg.a",
+    "-include-binary", "$rootDir/external/core/capi_macos_universal/src/realm/object-store/librealm-object-store-dbg.a"
 )
 // TODO remove the debug suffix (-dbg) when switching to use release
 val nativeLibraryIncludesIosArm64 = listOf(
-    "-include-binary", "$rootDir/../external/core/build-capi_ios_Arm64/lib/librealm-ffi-static-dbg.a",
-    "-include-binary", "$rootDir/../external/core/build-capi_ios_Arm64/lib/librealm-dbg.a",
-    "-include-binary", "$rootDir/../external/core/build-capi_ios_Arm64/lib/librealm-parser-dbg.a",
-    "-include-binary", "$rootDir/../external/core/build-capi_ios_Arm64/lib/librealm-object-store-dbg.a"
+    "-include-binary", "$rootDir/external/core/build-capi_ios_Arm64/lib/librealm-ffi-static-dbg.a",
```

Comment:
Any reason to not use the the `$corePath` variable here? so `$rootDir/$corePath`?

Reply:
The cinterop paths has to be absolute so they have to be handled differently. I have included another variable for the cinterop specific prefix.